cmd/monaco/integrationtest/account/all_resources_integration_test.go

```diff
@@ -42,7 +43,7 @@ func TestDeployAndDelete_AllResources(t *testing.T) {
 	cliDeployMZones.SetArgs([]string{"deploy", "testdata/all-resources/manifest-mzones.yaml"})
 
 	err := cliDeployMZones.Execute()
-	assert.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
Will this work for local execution during debugging (where the version is not set to 2.x)?

Reply:
I don't understand how this relates to the version being set?

Reply:
Sorry, wrongly reported...

Reply:
no problem :)